.github/workflows/build.yml

```diff
@@ -18,19 +18,15 @@ jobs:
           - os: ubuntu-latest
             sanitizer: address
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - name: Setup Rust (regular)
         if: matrix.sanitizer != 'address'
-        uses: actions-rs/toolchain@v1
-        with:
-          toolchain: 1.58.1
-          default: true
+        uses: dtolnay/rust-toolchain@1.58.1
       - name: Setup Rust (for sanitizer)
         if: matrix.sanitizer == 'address'
-        uses: actions-rs/toolchain@v1
+        uses: dtolnay/rust-toolchain@master
```

Comment:
why @master here?

Reply:
This is due to the use of the `nightly` toolchain - there's no specific `rev` for each version of nightly